mui/dockwidgets/state_list_widget.py

```diff
@@ -46,6 +49,30 @@ def __init__(self, name: str, parent: ViewFrame, data: BinaryView):
         self.states: Dict[int, StateDescriptor] = {}
         self.state_items: Dict[int, QTreeWidgetItem] = {}
 
+    @Slot(QTreeWidgetItem, int)
+    def onClick(self, item: QTreeWidgetItem, col: int):
+        """Jump to the current PC of a given state when double clicked"""
+
+        # do nothing on non-state items
+        if item in self.state_lists:
+            return
+
+        item_id = int(item.text(0).split(" ")[-1])
+        state = self.states[item_id]
+
+        if isinstance(state.pc, int):
+            self.bv.navigate(self.bv.view, state.pc)
+        elif isinstance(state.last_pc, int):
+            # use last_pc as a fallback
+            self.bv.navigate(self.bv.view, state.last_pc)
+        else:
+            show_message_box(
+                "[MUI] No instruction information available",
+                f"State {item_id} doesn't contain any instruction information.",
+                MessageBoxButtonSet.OKButtonSet,
+                MessageBoxIcon.ErrorIcon,
+            )
```

Comment:
I noticed this message box shows up for the `Errored` state in the `multiple-styles` example. That's fine for now, but I wonder if something can be done to record the instruction that caused the error or where it was when it errored.

We should create a new issue if this isn't easily resolvable or you don't want to resolve in this PR.

Reply:
It seems that only the `transition_state` event fires when manticore kills all the remaining states during exit, I changed the code to hook this event instead of `terminate_state` and `kill_state`. Now, all the errored states have their last instruction recorded correctly.

Reply:
Nice find!